Source/IGListCollectionContext.h

```diff
@@ -253,6 +253,20 @@ NS_ASSUME_NONNULL_BEGIN
                    scrollPosition:(UICollectionViewScrollPosition)scrollPosition
                          animated:(BOOL)animated;
 
+/**
+ Invalidates the collection view layout for all items in the given section.
+ 
+ @param sectionController The section controller.
+ */
+- (void)invalidateLayoutForSectionController:(IGListSectionController<IGListSectionType> *)sectionController;
+
+/**
+ Invalidates the collection view layout for the items at the given index paths.
+ 
+ @param indexPaths The index path array to invalidate.
+ */
+- (void)invalidateLayoutForIndexPaths:(NSArray<NSIndexPath *> *) indexPaths;
```

Comment:
Will reduce down to just a single API, if a user wants to reset the whole section then it's not that difficult to create an index set for the entire range